yaml/shared/src/main/scala/org/virtuslab/yaml/internal/load/reader/Reader.scala

```diff
@@ -21,15 +21,15 @@ trait Reader:
   def line: Int
   def column: Int
   def offset: Int
-  def pos: Position
+  def pos: Range
```

Comment:
```suggestion
  def pos: Position
  def range(start: Position = in.pos, end: Position = in.pos): Range
```
thus will allow:
```scala
val start = in.pos
// parse something
val range = in.range(start = start)
```
wdyt?